doc/sphinx/how-to/how-to-handle-user-input.md

```diff
@@ -0,0 +1,97 @@
+# How to Handle User Input
+
+**Note**: This builds on top of [Write Your First Wayland
+Compositor](../make-your-own-compositor.md).
```

Comment:
I think that the purpose of the howto should come first. Then we should state that we assume that the reader has completed the tutorial